src/main/java/org/aksw/iguana/cc/storage/impl/TriplestoreStorage.java

```diff
@@ -75,7 +79,11 @@ public void storeResult(Model data) {
 		//submit Block to Triple Store
 		UpdateProcessor processor = UpdateExecutionFactory
 				.createRemote(blockRequest, endpoint, createHttpClient());
-		processor.execute();
+		try {
+			processor.execute();
+		} catch (Exception e) {
+			logger.error("Error while storing data in triplestore: " + e.getMessage());
```

Comment:
The RDFFileStorage and CSVStorage are written in a similar way. Other than crashing, I wouldn't know what else should happen when the exception occurs. I added this, so that iguana doesn't just crash during the run, where graalvm generates the configurations.

Reply:
> I added this, so that iguana doesn't just crash during the run, where graalvm generates the configurations.

This refers to profiling for graalvm, right?

Reply:
Not for the profiling, but for when you execute the generate-config.sh script. The configuration generated by the tracing agent.

Reply:
But why should it crash?

Reply:
Because the script just tries to run every feature of iguana, but it doesn't actually benchmark a triplestore and there is also no triplestore where the data can be stored with the TriplestoreStorage. That's why it throws an exception at that point.

Reply:
I think it would be better if it had an commandline option that is not documented in `--help` that enables this behavior for GraalVM instead of making it default behavior.

Reply:
I doesn't seen to be possible to hide the option from the  `--help` documentation. Should I add it anyway?